apps/client/src/features/modals/EventSettingsModal.jsx

```diff
@@ -99,19 +103,19 @@ export default function SettingsModal() {
           </div>
           <div className={style.hSeparator}>Additional Screen Info</div>
           <div className={style.spacedEntry}>
-            <FormLabel htmlFor='url'>
-              Event URL
+            <FormLabel htmlFor='pubUrl'>
```

Comment:
Nitpick nr 1: I think we should stick to the same naming convention in each entry. So `publicUrl` here and in the name of the input